### ISSUE TITLE
Replacement for UI Module

### ISSUE THREAD

**Issue by gerbilOFdoom** [JKGDevs/JediKnightGalaxies#77]
In order to extend JKG to its fullest potential after Versus, the UI module needs to be replaced. After discussing with eezstreet it is believed that Awesomium is the way to go. We can integrate a web-browser overlay into the game so that every UI function is done via JQuery/HTML. The result will be user interfaces that can be customized per-server, per-person, and even per-level or level area. Beyond the UI functionality Awesomium presents, web pages can be easily integrated into servers to allow more advanced functionality such as buying/selling while not ingame, chatting, and even influencing the flow of server events.

tl;dr: We know JQuery, let's use JQuery, MOAR JQuery.

**Comment by eezstreet** [comment]
I've decided that this isn't something that we're going to do. The UI system is already pretty workable, and we'd have to rewrite a lot of the menus.

**Comment by DarthFutuza** [comment]
Sounds fair, probably more work than worth the benefit.
